MagnetScannerIOS/ScannerGeolocation.swift

```diff
@@ -10,88 +10,36 @@ import CoreLocation
 import SwiftyJSON
 
 class ScannerGeolocation: NSObject, CLLocationManagerDelegate, Scanner {
-  let locationManager: CLLocationManager = CLLocationManager()
+  var locationResolver: OneShotLocation = OneShotLocation()
```

Comment:
I find the naming `OneShotLocation` quite confusing, what is it implying? From a consumer standpoint, `getLocation` seems to be clearer naming to me. Unless I've missed something :)